core/src/display_object/movie_clip.rs

```diff
@@ -3962,29 +3962,18 @@ impl<'gc, 'a> MovieClipData<'gc> {
         context: &mut UpdateContext<'_, 'gc>,
         reader: &mut SwfStream<'a>,
     ) -> Result<(), Error> {
-        tracing::warn!("DefineFont4 tag (TLF text) is not implemented");
         let font = reader.read_define_font_4()?;
         let font_id = font.id;
-        let font_object = Font::from_swf_tag(
+        let font_object = Font::from_font4_tag(
             context.gc_context,
-            context.renderer,
-            swf::Font {
-                version: 4,
-                id: font.id,
-                name: SwfStr::from_bytes(b"<Unknown Font4>"),
-                language: swf::Language::Unknown,
-                layout: None,
-                glyphs: Vec::new(),
-                flags: FontFlag::empty(),
-            },
+            font,
             reader.encoding(),
             FontType::EmbeddedCFF,
```

Comment:
This should always be `EmbeddedCFF`, can you hard-code it to be that in `from_font4_tag`?